libs/components/modals/src/lib/modules/confirm/confirm.service.ts

```diff
@@ -24,8 +23,8 @@ export class SkyConfirmService {
   constructor(private modalService: SkyModalService) {}
 
   /**
-   * Opens a confirmation dialog and passes the `config` parameter of type `SkyConfirmConfig`
-   * to the component's constructor. This method returns an object of type `SkyConfirmInstance`.
+   * Opens a dialog using the specified options.
+   * @param config Specifies configuration options for the confirmation dialog.
```

Comment:
```suggestion
   * @param config Specifies configuration options for the dialog.
```
We delete "confirmation" elsewhere, so should we delete it here too?